R/pkg/R/DataFrame.R

```diff
@@ -3715,3 +3715,34 @@ setMethod("rollup",
             sgd <- callJMethod(x@sdf, "rollup", jcol)
             groupedData(sgd)
           })
+
+#' hint
+#'
+#' Specifies execution plan hint on the current SparkDataFrame.
```

Comment:
ouch sorry I didn't mean to put ".... hint and return a new SparkDataFrame"
but "Specifies execution plan hint and return a new SparkDataFrame"